lib/ruby_lsp/requests/folding_ranges.rb

```diff
@@ -203,6 +215,7 @@ def on_call(node)
           return
         end
 
+        emit_requires_range
```

Comment:
Based on the old implementation: `emit_requires_range unless node.is_a?(YARP::CallNode)`, I thought call node should be skipped?